core/src/main/java/org/commonjava/maven/ext/core/state/RESTState.java

```diff
@@ -84,6 +92,8 @@ public void initialise( Properties userProps )
         restSuffixAlign = Boolean.parseBoolean( userProps.getProperty( REST_SUFFIX, "true" ) );
 
         String repositoryGroup = userProps.getProperty( REST_REPO_GROUP, "" );
+        Boolean brewPullActive = BooleanUtils.toBooleanObject( userProps.getProperty( REST_BREW_PULL_ACTIVE ) );
+        Boolean temporaryBuild = BooleanUtils.toBooleanObject( userProps.getProperty( REST_TEMPORARY_BUILD ) );
```

Comment:
Why not use `Boolean.parseBoolean` rather than an external library?

Reply:
Because the logic using the flags also need to know if the value was passed in or not (using BooleanUtils.toBooleanObject(null) results in null). There is no such method in Boolean class AFAIK.

Reply:
Huh, is the boolean being used as a tri-state value ?